ontology/yaml/resources/ELECTRICAL/entity_types/UPS.yaml

```diff
@@ -68,8 +68,9 @@ UPS_IOBM_PPWM_UPSBM:
   - UPSBM
 
 UPS_PWM:
+  id: "4040385961387884544"
```

Comment:
these are going away, not sure if you want to wait until the update?
@jmact17 wdyt?

Reply:
I think it's ok to update this for now, the guid backfill tool will replace everything